pkg/util/os/atexit/atexit.go

```diff
@@ -29,6 +31,12 @@ func Exit(code int) {
 	os.Exit(code)
 }
 
+// ExitWithMessage exit and print the error message.
+func ExitWithMessage(code int, msg string) {
+	fmt.Println(msg)
```

Comment:
I suggest one more change, namely that we change the output steam depending on the exit code. If the exit code is '0' we log to stdout and if it is not equal '0' we print to stderr. We can use _fmt.Fprintln_ for that, selecting the right stream.